drivers/include/tmp006.h

```diff
@@ -101,6 +101,20 @@ extern "C"
 #endif
 
 /**
+ * @brief   Default low power mode
```

Comment:
Change this to describe what the option controls instead of the line itself. Something like
```
Low power operation

If set to 1, the device will be put in low power mode between measurements. This adds a @c TMP006_CONVERSION_TIME us delay to each measurement call for bringing the device out of standby. 

```